demo/tests/test_end_point.py

```diff
@@ -328,6 +328,62 @@ class TestForm(FormidableForm):
         with django_perf_rec.record(path='perfs/'):
             serializer.data
 
+    def test_presets(self):
+
+        form = Formidable.objects.create(label='my_test_form')
```

Comment:
Okay, let me introduce you `FormidableForm` , our Python Builder in order to generate Formidable objects.

I don't know how you are familiar with Django Form, but Formidable project provides a way to generate data test easily. In order to have more consistent data, I suggest to have a Form with appropriate field which actually match your presets Arguments.

So, you can start to import (if its not the case), 

```
from formidable.forms import FormidableForm, fields
```
Begin with declaring a new form in your method, and add appropriate fields.

```python

class MyAwesomeForm(FormidableForm):
     Field_A = fields.IntegerField()

``` 

Then you are able to generate the appropriate Formidable objects

```python

formidable = MyAwesomeForm.to_formidable(label='my_test_form')

```

Now you can plug the validations presets in it.